packages/compass-web/src/index.tsx

```diff
@@ -101,6 +101,7 @@ type CompassWorkspaceProps = Pick<
 
 type CompassWebProps = {
   darkMode?: boolean;
+  stackedElementZIndex?: number;
```

Comment:
Since this is optional I'm wondering if we should set it to something in the compass web sandbox just so it is set at least once in our codebase and then we can test it and encounter the code paths where it is set. ie. at least in the compass web sandbox' e2e tests.